.github/workflows/cmd-tests.yml

```
@@ -11,7 +11,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  test-cmd:
```

```suggestion
  test-cmd-bot:
```
I would use this name because `test-cmd` is still confusing imo